sdk/core/azure-core-amqp/vcpkg/vcpkg.json

```diff
@@ -15,7 +15,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.9.0-beta.1"
+      "version>=": "1.10.0"
```

Comment:
If you do this because it didn't work for curl, it is not required, although it is not a bad change.

Curl has `version-string`, Azure SDK libraries have `version-semver` - which means that they always can be ordered even without knowing the full set of versions. I was a bit surprised that vcpkg dind't handle it well for curl version - even though it was a `version-string`, `8.44` comes perfectly before `8.88` (or whatever the version was) using the lexicographical compare.

If we _have_ to do this for `version-semver`s as well, that greatly reduces the usability of private registries. We will always have to keep in mind what versions we are publishing and which dependencies are published where, and update them accordingly. I'd say, at that point, it would make sense to _consider_ dropping `version>=` alltogether - that's how unusable it is in vcpkg then.

If it is not the case, I think it would make sense to suggest libcurl maintainers to switch versioning schema in vcpkg to `version-semver`.

Reply:
BTW, if you want to update this in vcpkg without re-releasing, there is a way - use `port-version: 1`, i.e. pretty much how Billy have done it.

Reply:
AMQP is basically working,

Core is still horribly broken. And curl uses version-simple not version-string.

When you come back next week (this is what is known as a "hint" - you're on vacation :)), I'll explain the whole horrible story. It's very icky.

Reply:
We fixed the AMQP package here: https://github.com/microsoft/vcpkg/pull/33176

Reply:
Ouch, that's pretty bad that vcpkg has this limitation. Let's discuss next week. Thank you guys for looking into this!